sky/shell/tracing_controller.h

```diff
@@ -50,25 +45,24 @@ class TracingController : public mojo::common::DataPipeDrainer::Client {
   bool picture_tracing_enabled() const { return picture_tracing_enabled_; }
 
  private:
-  std::unique_ptr<mojo::common::DataPipeDrainer> drainer_;
   std::unique_ptr<base::File> trace_file_;
-  // TODO: Currently, only the last shell view is traced. When the shell gains
-  // the ability to host multiple shell views, references to each must be stored
-  // instead and trace data from each serialized to the output trace.
-  ShellView* view_;
-  base::FilePath picture_tracing_base_path_;
+  base::FilePath traces_base_path_;
   bool picture_tracing_enabled_;
+  bool terminate_loop_on_write_;
```

Comment:
It's strange that this bool is global to the tracing controller, but I guess that's because the tracing data is held in statics anyway so you can't really have two trace collection jobs going on at once.